sqlelf/elf.py

```diff
@@ -35,7 +36,40 @@ def make_generator(
         return Generator(columns, apsw.ext.VTColumnAccess.By_Name, generator)
 
 
-def make_dynamic_entries_generator(binaries: list[lief.Binary]) -> Generator:
+class GeneratorFlag(Flag):
+    NONE = 0
+    DYNAMIC_ENTRIES = auto()
+    HEADERS = auto()
+    INSTRUCTIONS = auto()
+    SECTIONS = auto()
+    SYMBOLS = auto()
+    STRINGS = auto()
+    VERSION_REQUIREMENTS = auto()
+    VERSION_DEFINITIONS = auto()
+
+    @classmethod
+    def ALL(cls: type[GeneratorFlag]) -> GeneratorFlag:
+        retval = cls.NONE
+        for member in cls.__members__.values():
+            retval |= member
+        return retval
+
+
+@dataclass
+class MakeGeneratorResponse:
```

Comment:
@markrwilliams maybe you have ideas how to make this nicer.
Kind of hate my naming and how this all gets constructed....
at least it's data class /shrug